### PR TITLE
Feat/implement heat trace app

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**

### CODE REVIEW

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

**Comment by MagnusGjosund** [comment]
deploy test

File: apps/heattrace/app.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const { name } = JSON.parse(readFileSync('./package.json').toString('utf-8'));
+
+export default () => ({
+  manifest: {
+    key: name,
+    name: name,
+  },
+  environment: {
+    // uri: 'https://backend-fusion-data-gateway-test.radix.equinor.com',
+    uri: 'https://localhost:7074',
```
Comment (Gustav-Eikaas) [inline apps/heattrace/app.config.js:12]: fix

File: libs/heattraceapp/src/lib/config/frameworkConfig.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          },
+        },
+      });
+    });
+  });
+
+  const envConfig: HeattraceEnvConfig = c.env.config?.environment as HeattraceEnvConfig;
+
+  if (!envConfig) {
+    throw new Error('Failed to load environemnt config for workorder');
```
Comment (Gustav-Eikaas) [inline libs/heattraceapp/src/lib/config/frameworkConfig.ts:27]: typo, assuming this was copy paste so maybe we should fix more places

File: libs/heattraceapp/src/lib/config/tableConfig.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    },
+  },
+  {
+    colId: 'HeatTraceCableDescription',
+    field: 'Description',
+    valueGetter: (pkg) => pkg.data?.heatTraceCableDescription,
+    cellRenderer: (props: ICellRendererProps<HeatTrace, string | null>) => {
+      return <DescriptionCell description={props.value} />;
+    },
+    width: 300,
```
Comment (Gustav-Eikaas) [inline libs/heattraceapp/src/lib/config/tableConfig.tsx:63]: no widths!

File: libs/heattraceapp/src/lib/config/tableConfig.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  },
+  {
+    field: 'Pipetests',
+    valueGetter: (pkg) => pkg.data?.pipetest,
+    cellRenderer: (props: ICellRendererProps<HeatTrace, string>) => {
+      if (!props.value) return null;
+      return <StyledMonospace>{props.value}</StyledMonospace>;
+      // (
+      //   <StyledMonospace>
+      //     {generateCommaSeperatedStringArrayColumn(getHTList(props.value))}
```
Comment (Gustav-Eikaas) [inline libs/heattraceapp/src/lib/config/tableConfig.tsx:92]: Remove unused comments??

Comment (MagnusGjosund) [reply libs/heattraceapp/src/lib/config/tableConfig.tsx:92]: its code that will be uncommented when i recive more data fields

Comment (Gustav-Eikaas) [reply libs/heattraceapp/src/lib/config/tableConfig.tsx:92]: I remain skeptical

**Comment by MagnusGjosund** [comment]
deploy test

**Pull request merged**